src/content/docs/guides/platform-setup/configure-internal-services.mdx

```diff
@@ -13,7 +13,9 @@ configuration parameters from environment variables.
 Create a file named `.env` in the same directory as your `docker-compose.yaml`
 file and set the environment variables described below.
 
-## Admin Rules
+## Tenzir Platform
```

Comment:
Everything in this section is about Tenzir Platform. Can we find something more specific?